jina/drivers/control.py

```diff
@@ -112,6 +100,38 @@ def __call__(self, *args, **kwargs):
         else:
             raise UnknownControlCommand(f'don\'t know how to handle {self.req.command}')
 
+    def _reload(self):
+        # TODO should this be removed, since we now have proper rolling update?
```

Comment:
Let's wait, I think some users rely on this for now

Reply:
sure